packages/cmd/src/help/WebHelpGenerator.ts

```diff
@@ -320,17 +315,13 @@ export class WebHelpGenerator {
             { commandDefinition: definition, fullCommandTree: this.mFullCommandTree, skipTextWrap: true });
 
         let markdownContent = helpGen.buildHelp() + "\n";
-        markdownContent = markdownContent.replace(/</g, "&lt;").replace(/>/g, "&gt;");
```

Comment:
This "hack" is no longer needed, since `markdown-it` handles angle brackets correctly. It seems to have been left behind as a remnant from when we were using `marked`.